metabrainz/templates/users/signup-commercial.html

```diff
@@ -77,8 +77,8 @@ <h1 class="page-title">Sign up <small>Commercial</small></h1>
       <div class="col-sm-5">{{ form.logo_url(class="form-control", placeholder="(preferably in SVG format)") }}</div>
```

Comment:
I fixed this in one of the commits, but removed it later as using the gettext function is giving problems like these- http://imgur.com/a/2p42w

possible reason is nesting of blocks, working on a fix